app/components/ui/PeoplePicker/PeoplePickerBody.js

```diff
@@ -26,7 +37,14 @@ const SelectedItems = ({ selection, onCloseClick }) => (
   </SelectedItemsGrid>
 )
 
-const SuccessMessage = styled.div`
+const MessageWrapper = styled.div`
+  ${media.tabletPortraitUp`
+    position: absolute;
+    top: -60px;
```

Comment:
@Cooryd and I had the discussion about whether this is a multiple of the grid size... but its not so we are leaving it a magic number.